android/java/res/layout/brave_wallet_accounts_list_item.xml

```diff
@@ -55,6 +55,18 @@
        android:id="@+id/accountCheck"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
+       android:visibility="gone"
        android:layout_gravity="end" />
 
+    <TextView
+        android:id="@+id/accountAction"
```

Comment:
nit : naming convention is different than `android:id="@+id/sub_title"`

Reply:
done